### DISCUSSION TITLE
Declaring a public tuple auto-implemented property in the class interface

### DISCUSSION THREAD

**Discussion by ebonat** [dotnet/csharplang#3932] [General]
hi,

i have a tuple auto-implemented property in a class as example:

public double (double a, double b, double c) Number {get, set};

what is the best practice to define this property in the class interface? i want a client program to access the Number tuple property.

thank you for your help,

**Comment by CyrusNajmabadi** [comment]
`interface IFoo { (double a, double b, double c) Number { get; set; } }`
